include/can_interface.h

```diff
@@ -313,16 +403,17 @@ class ICANTXMessage
 {
 public:
     virtual uint32_t GetID() = 0;
+#if !defined(NATIVE)  // workaround for unit tests
```

Comment:
this is still giving me testing errors when we attempt to override the function later: 
```
include/can_interface.h:612:38: error: only virtual member functions can be marked 'override'
    VirtualTimer &GetTransmitTimer() override { return transmit_timer_; }
                                     ^~~~~~~~~
include/can_interface.h:926:38: error: only virtual member functions can be marked 'override'
    VirtualTimer &GetTransmitTimer() override { return transmit_timer_; }
```

Reply:
This should be fixed now